src/DynamoCore/Graph/Workspaces/Serialiaztion/Workspaces.cs

```diff
@@ -83,6 +47,11 @@ public static class Utilities
             return ws;
         }
 
+        public static string SaveWorkspaceToJson(DynamoModel model)
```

Comment:
I don't like this - it should be clear what workspace is getting saved, can you move the WS as a parameter of the function. What if I want to save a custom node.